README.md

```diff
@@ -1,99 +1,88 @@
+
+To use this app, you need to add it in your `dependencies` in the `manifest.json` file.
+
+```json
+  dependencies: {
+    "vtex.carousel": "2.x"
+  }
 ```
 
-### Running Tests
+Then, add the `carousel` block into our app theme, as we do in our [Dreamstore app](https://github.com/vtex-apps/dreamstore/blob/master/store/blocks.json). 
+
+### Blocks API
 
-To run the test suit, type the following in the terminal, inside the folder `react`
+This app has an interface that describes what rules must be implemented by a block when you want to use the carousel app.
 
-```sh
-$ npm t     # this
+```json
+{
+  "carousel": {
+    "component": "Carousel"
+  }
+}
 ```
 
+#### Configuration 
+Through the Storefront, you can change the behavior and interface of the carousel. However, you also can make in your theme app, as Dreamstore does.
+
+| Prop name          | Type       | Description                                                                 |
+| ------------------ | ---------- | --------------------------------------------------------------------------- |
+| `autoplay`                  | `Boolean`      | Enable automatic banner transition                                 |
+| `autoplaySpeed`             | `Number`       | Set the automatic banner transition interval                       |
+| `showDots`                  | `Boolean`      | Shows the carousel dots                                            |
+| `showArrows`                | `Boolean`      | Shows the carousel arrows                                          |
+| `height`                    | `Number`       | Set banners height                                                 |
+| `banners`                   | `Array(Banner)`| Array of banners the will be shown in the carousel                 |
+
+Banner:
+
+| Prop name          | Type       | Description                                                                 |
+| ------------------ | ---------- | --------------------------------------------------------------------------- |
+| `image`                     | `String`      | Link for the image of the banner                                   |
+| `mobileImage`               | `String`      | Link for the mobile image of the banner                            |
+| `description`               | `String`      | The image's description                                            |
+| `url`                       | `String`      | The URL where the image is pointing to, in case of external route  |
+| `page`                      | `String`      | The page where the image is pointing to                            |
+| `params`                    | `String`      | Parameters of the URL                                              |
+| `externalRoute`             | `Boolean`     | Indicates if the route is external or not                          |
+
+### Styles API
+:construction: :construction: :construction:
```

Comment:
Update the styles section

Reply:
It is construction because we are waiting for the Slider API.

Reply:
You can add the initial content of the section, like on minicart app